gitversion.yml

```diff
@@ -0,0 +1,21 @@
+assembly-versioning-scheme: MajorMinorPatch
+mode: ContinuousDelivery
+next-version: 0.9.1
```

Comment:
The purpose for this: https://gitversion.net/docs/reference/version-increments?#gitversion.yml